conn.go

```diff
@@ -195,6 +196,7 @@ func (conn *Conn) Close() error {
 		close(conn.eavesdropped)
 	}
 	conn.eavesdroppedLck.Unlock()
+	conn.outWait.Wait()
```

Comment:
Oh, sorry. I didn't look at the code just the context, so somehow i thought the channel would be written to from the worker, not read.